src/allmydata/util/idlib.py

```diff
@@ -14,15 +14,15 @@
 from foolscap import base32
 
 
-def nodeid_b2a(nodeid):
+def nodeid_b2a(nodeid: str) -> str:
```

Comment:
Would nodeid make a good type?

Reply:
What library would that be from? I think nodeid should also be bytes right? I understand we want stricter type annotations and I think bytes might be more appropriate, since the function changes bytes to ASCII.